setup.py

```diff
@@ -29,7 +29,7 @@
     },
     license="MIT",
     python_requires=">=3.7",
-    packages=find_namespace_packages(include=["geoalchemy2"]),
+    packages=find_namespace_packages(include=["geoalchemy2", "geoalchemy2.dialects"]),
```

Comment:
```suggestion
    packages=find_namespace_packages(include=["geoalchemy2*"]),
```
It would be simpler to write this since we also have to add `geoalchemy2.admin`, `geoalchemy2.admin.dialects`, `geoalchemy2.types` and `geoalchemy2.types.dialects`.